core/primitives-core/src/config.rs

```diff
@@ -5,6 +5,7 @@ use std::collections::hash_map::DefaultHasher;
 use std::hash::{Hash, Hasher};
 use strum::{Display, EnumCount};
 
+/// Describes сonfiguration of wasm operations.
```

Comment:
Configuration might be misleading. What is described here is cost of different operations